cmd/kaniko-docker/main.go

```diff
@@ -448,6 +453,7 @@ func run(c *cli.Context) error {
 			SkipTLSVerifyRegistry:       c.Bool("skip-tls-verify-registry"),
 			UseNewRun:                   c.Bool("use-new-run"),
 			IgnorePath:                  c.String("ignore-path"),
+			LocalTarPath:                c.String("local-tar-path"),
```

Comment:
Place options/fields in an alphabetical order.

Reply:
Nit: `ImageTarPath` ?

Reply:
Changed `LocalTarPath` to `ImageTarPath` and arranged them in alphabetical order. ✅